src/error/mod.rs

```diff
@@ -156,3 +156,15 @@ impl<E: Into<anyhow::Error>> From<E> for RoverError {
         Self::new(error)
     }
 }
+
+impl From<RoverError> for BuildErrors {
```

Comment:
maybe `TryFrom` instead of `From` to avoid the panic and let the caller deal with the error?